pom.xml

```diff
@@ -112,6 +112,9 @@
         <opentracing-kafka.version>0.1.13</opentracing-kafka.version>
         <strimzi-oauth.version>0.6.1</strimzi-oauth.version>
         <commons-codec.version>1.13</commons-codec.version>
+        <registry.version>1.3.0.Final</registry.version>
+        <kafka.streams.version>2.6.0</kafka.streams.version>
```

Comment:
Does this have to be 2.6.0? Or can this be 2.7.0 now? It seems a bit weird that we use 2.7.0 for client but 2.6.0 for Streams. I'm not sure how Streams 2.6 deals with newer KAfka clients.

Reply:
This is what's used in Registry, via that utils-streams dependency ....
We can try to upgrade and see if it all works out.